plugins/content_distribution/providers/tvinci/lib/admin/TvinciProfileConfiguration.php

```diff
@@ -110,13 +118,11 @@ protected function tvinciElements()
 		);
 
 		// tag specific configuration
-		$this->addTagItems("ism");
-		$this->addTagItems("ipadnew");
-		$this->addTagItems("iphonenew");
-		$this->addTagItems("mbr");
-		$this->addTagItems("dash");
-		$this->addTagItems("widevine");
-		$this->addTagItems("widevine_mbr");
+		$tvinciDistributionTagsSubForm = new Zend_Form_SubForm(array('DisableLoadDefaultDecorators' => true));
```

Comment:
This happens twice once in the populateFromObject function and once here 
is this a different load path and if so - create a func that does this